jovo-platforms/jovo-platform-facebookmessenger/src/core/MessengerBotResponse.ts

```diff
@@ -42,7 +43,15 @@ export class MessengerBotResponse implements JovoResponse {
   }
 
   getSpeechPlain(): string | undefined {
-    return this.message instanceof TextMessage ? this.message.message.text : undefined;
+    return this.messages
```

Comment:
Upon updating messages to be an array, I realised this method needed to change to reflect the new structure. I wasn't too sure what to do about this method but thought it might be good to return all FB texts as a string separated by space. What's your opinion on this?

Reply:
Good catch! Yeah, I think that's good and should be sufficient. We will just have to update the docs that people know what to expect there.

Reply:
That's a good point, I can add that in for sure. How would you prefer we add that configuration? I was thinking for simplicity's sake, we could just add a new attribute in the `MessengerBotResponse` class or we could update the `getSpeechPlain` signature to take a separator input?

Reply:
I think being able to pass it as a parameter to `getSpeechPlain()` and making that parameter optional is probably the best option.

Reply:
Sounds good, I've updated the PR.

Reply:
Thanks you! I will check the changes once more and then we should be able to merge.